webpack/assets/javascripts/react_app/components/SettingsTable/components/SettingName.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SettingName = ({ setting }) => (
+  <React.Fragment>{setting.fullName}</React.Fragment>
```

Comment:
Alternatively to ^^ we could
```suggestion
  <React.Fragment>{setting.fullName ? __(setting.fullName) : setting.name}</React.Fragment>
```